CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+Version 1.11.2
+==============
+
+This is a minor bug fix release with a number of minor but important
+bug fixes. Special thanks to @darynwhite for his contributions.
+
+Bug fixes:
+
+- Fixed persisting options during clone on Overlay
+  ([#3435](https://github.com/ioam/holoviews/pull/3435))
+- Ensure cftime datetimes are displayed as a slider
+  ([#3413](https://github.com/ioam/holoviews/pull/3413))
+- Fixed timestamp selections on streams
+  ([#3427](https://github.com/ioam/holoviews/pull/3427))
+- Compatibility with pandas 0.24.0 release
+  ([#3433](https://github.com/ioam/holoviews/pull/3433))
```

Comment:
I'd move the pandas changes to the top; I think those are the ones people need to notice the most.